keyboards/rama/m60_a/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+// M60-A layout
+
+#include "m60_a.h"
```

Comment:
This should be changed to `#include QMK_KEYBOARD_H`.

Reply:
fixed